content/en/armory-enterprise/armory-admin/terraform-enable-integration.md

```diff
@@ -356,24 +356,24 @@ After you finish your Terraform integration configuration, perform the following
 
 ## Configure Terraform for your cloud provider
 
-Since the Terraform Integration executes all Terraform commands against the `terraform` binary, all methods of configuring authentication are supported for your desired cloud provider. This section describes how to accomplish this for various cloud providers.
+>Terraform Integration is cloud provider agnostic. Terraform commands execute against the `terraform` binary. All methods of configuring authentication are supported for your desired cloud provider.
```

Comment:
Terraform Integration is cloud provider agnostic. Terraform commands execute against the `terraform` binary. All methods of configuring authentication are supported as per Terraform's compatibility and capabilities.